dev-python/wrapt/wrapt-1.12.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Python module for decorators, wrappers and monkey patching"
+DESCRIPTION="The aim of the wrapt module is to provide a transparent object \
+proxy for Python, which can be used as the basis for the construction of \
+function wrappers and decorator functions."
+HOMEPAGE="https://github.com/GrahamDumpleton/wrapt/archive/1.12.1.tar.gz"
+COPYRIGHT="2013-2019, Graham Dumpleton"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/GrahamDumpleton/$portName/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="19f2043e05b76ce9b68971b387bfd8eed50c2ee1c9c0bd230edc422fec753917"
+SOURCE_DIR="$portVersionedName"
```

Comment:
Shouldn't it be "wrapt-$portVersion"? @Begasus?

Reply:
As this is "any" architecture it should be ok (need to check), if this was meant for secondary architecture it wont work (renames the source dir to wrapt_x86 then